### PR TITLE
Update subscribe

### PR DESCRIPTION
**Pull request by marianzburlea**
this is related to issue #285

### CODE REVIEW

**Review by tandaraelena — APPROVED** [review]
:+1:

File: src/component/_dumb/button/button.style.js [hunk 1]
```diff
@@ -18,6 +19,10 @@ export const StyledButton = styled.button`
   display: inline-flex;
   align-items: center;
 
+  &:disabled {
+    cursor: not-allowed;
```
Comment (BogdanNegura) [inline src/component/_dumb/button/button.style.js:23]: :+1:

**Review by BogdanNegura — APPROVED** [review]
:100:

**Review by robertoalx — APPROVED** [review]
💯 👍

**Review by ghost — APPROVED** [review]
👍

**Pull request merged**